packages/engine/Source/Scene/Model/MaterialPipelineStage.js

```diff
@@ -216,6 +216,44 @@ function processTextureTransform(
   };
 }
 
+/**
+ * Process a single texture scale and add it to the shader and uniform map.
+ *
+ * @param {ShaderBuilder} shaderBuilder The shader builder to modify
+ * @param {Object<string, Function>} uniformMap The uniform map to modify.
```

Comment:
```suggestion
 * @param {Object<string, function>} uniformMap The uniform map to modify.
```

Reply:
I see your point here, we normally don't use the capitalized names for primitive types. However, in the case of `function`, I think we are capitalized everywhere. Apparently this wasn't addressed in https://github.com/CesiumGS/cesium/pull/11080.
Do we want to start fixing these? Or wait for https://github.com/CesiumGS/cesium/issues/10815?

Reply:
Gotcha, my bad. We can hold off until #10815.